doc/ChangeLog

```diff
@@ -1,4 +1,124 @@
 ===============================================================
+Tag name:  ctsm5.1.dev020
+Originator(s):  slevis (Samuel Levis,303-665-1310)
+Date:  Wed Dec 23 11:29:33 MST 2020
+One-line Summary: Grid cell-level error check for H2O
+
+Purpose of changes
+------------------
+
+ For more robust mass balance error checking, introduced
+ grid cell-level error check for H2O following the approach
+ of pull requests #984 and #1022
+
+
+Bugs fixed or introduced
+------------------------
+
+Issues fixed (include CTSM Issue #): #201
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[X] clm5_1
+
+[X] clm5_0
+
+[X] ctsm5_0-nwp
+
+[X] clm4_5
```

Comment:
I am realizing that I should have considered the rms diff (not the normalized diff), and I should remove the Xs.

Reply:
DONE